build/config/build.yaml

```diff
@@ -82,4 +87,61 @@ steps:
   dir: 'tests/'
   waitFor: ['DOCKER_STAGE', 'DOCKER_BUILD_MVN_GCLOUD']
 
+#
+# End of App Engine Integration tests
+
+#
+# GKE Integration tests
+#
+
+# Create a Docker image with the test application
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'DOCKER_BUILD_TEST_APPLICATION'
+  args: ['build', '--tag=${_STAGING_IMAGE}-test-app', '--no-cache', 'tests/runtimes-common-testing/target/deploy']
```

Comment:
I think the issue is that the `JETTY_SMOKE_TEST` step also uses other build resources that are produced during other parts of the maven build. Maybe the best solution is just for `JETTY_SMOKE_TEST` not to reach into other modules' target directories.